egs2/voxceleb/spk1/conf/tuning/train_RawNet3_sgdr.yaml

```diff
@@ -0,0 +1,62 @@
+# RawNet3 reproduce recipe configuration.
+
+frontend: raw
+
+encoder: rawnet3
+encoder_conf:
+  model_scale: 8
+  ndim: 1024
+  sinc_stride: 16
+
+pooling: chn_attn_stat
+pooling_conf:
+  input_size: 1536  # 1.5 * ndim of RawNet3 encoder
+
+projector: rawnet3
+projector_conf:
+  input_size: 3072  # 2 * input_size of pooling
+  output_size: 256
+
+preprocessor: spk
+preprocessor_conf:
+  utt2spk: dump/raw_copy/voxceleb12_devs/utt2spk
+  spk2utt: dump/raw_copy/voxceleb12_devs/spk2utt
+  target_duration: 3.0  # seconds
+  sample_rate: 16000
+  num_eval: 5
+
+model_conf:
+  extract_feats_in_collect_stats: false
+
+loss: aamsoftmax
+loss_conf:
+  nOut: 256
+  nClasses: 7205
```

Comment:
It is OK to keep arguments like the current format to keep it consistent with the source codebase, but is it possible to make consistent to other ESPnet style module? (making it to output_size and class_num

Reply:
I agree.
It would be better to unify it.
At least, it should be consistent with the config variable naming convention (lower case, "_" for the space)

Reply:
done!